server/index.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I think pretty much everything here should be added into app.js. It looks like app.js is only for maintaining sessions and there is nothing about sessions in index.js. We will be, however, editing line 66 of `app.use('/api/v0/posts', require('./api/posts'));`